addons/payment_authorize/controllers/main.py

```diff
@@ -11,21 +11,36 @@ class AuthorizeController(http.Controller):
 
     @http.route('/payment/authorize/get_acquirer_info', type='json', auth='public')
     def authorize_get_acquirer_info(self, acquirer_id):
-        acquirer_sudo = request.env['payment.acquirer'].sudo().browse(acquirer_id)
+        """ Return information on the acquirer.
+
+        :param int acquirer_id: The acquirer handling the transaction, as a `payment.acquirer` id
+        :return: Information on the acquirer, namely the state, login ID and client key
+        :rtype: dict
+        """
+        acquirer_sudo = request.env['payment.acquirer'].sudo().browse(acquirer_id).exists()
         return {
             'state': acquirer_sudo.state,
             'login_id': acquirer_sudo.authorize_login,
             'client_key': acquirer_sudo.authorize_client_key,
         }
 
-    @http.route('/payment/authorize/payments', type='json', auth='public')
-    def authorize_payments(self, opaque_data, reference, access_token, partner_id):
+    @http.route('/payment/authorize/payment', type='json', auth='public')
+    def authorize_payment(self, reference, partner_id, access_token, opaque_data):
+        """ Make a payment request and handle the response.
+
+        :param str reference: The reference of the transaction
+        :param int partner_id: The partner making the transaction, as a `res.partner` id
+        :param str access_token: The access token used to verify the provided values
+        :param dict opaque_data: The payment nonce provided by Authorize.Net
```

Comment:
nonce ?

Reply:
That's the name Authorize use for the single-use payment token in their doc :grimacing: 

https://developer.authorize.net/api/reference/features/acceptjs.html

Reply:
Yup, Antoine just explained it to me (and we replaced it by a more generic naming in the code ^^).